From 87c5c4ec5bee02725cb4bd4f81bb3c86a0277a81 Mon Sep 17 00:00:00 2001
From: hjl <hjl@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Wed, 30 Mar 2011 10:43:55 +0000
Subject: [PATCH 043/200] Fix a typo in FLOAT_SSE_REGS.

2011-03-30  H.J. Lu  <hongjiu.lu@intel.com>

	Backport from mainline
	2011-03-30  H.J. Lu  <hongjiu.lu@intel.com>

	PR target/48349
	* config/i386/i386.h (REG_CLASS_CONTENTS): Fix a typo in
	FLOAT_SSE_REGS.


git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_6-branch@171719 138bc75d-0d04-0410-961f-82ee72b054a4

index f14a95d..981052f 100644
--- a/gcc/config/i386/i386.h
+++ b/gcc/config/i386/i386.h
@@ -1272,7 +1272,7 @@ enum reg_class
 { 0xe0000000,    0x1f },		/* MMX_REGS */			\
 { 0x1fe00100,0x1fe000 },		/* FP_TOP_SSE_REG */		\
 { 0x1fe00200,0x1fe000 },		/* FP_SECOND_SSE_REG */		\
-{ 0x1fe0ff00,0x3fe000 },		/* FLOAT_SSE_REGS */		\
+{ 0x1fe0ff00,0x1fe000 },		/* FLOAT_SSE_REGS */		\
    { 0x1ffff,  0x1fe0 },		/* FLOAT_INT_REGS */		\
 { 0x1fe100ff,0x1fffe0 },		/* INT_SSE_REGS */		\
 { 0x1fe1ffff,0x1fffe0 },		/* FLOAT_INT_SSE_REGS */	\
-- 
1.7.0.4

